packages/widgets/src/components/AccountBalance/BalancePieChart.tsx

```diff
@@ -0,0 +1,89 @@
+import { withTheme, css } from '@emotion/react';
+import { MainColors } from '@ssa-ui-kit/core';
+
+import { PieChart, PieChartLegend } from '@components/PieChart';
+import { BalancePieChartTitle } from './BalancePieChartTitle';
+
+import { BalancePieChartProps } from './types';
+
+const colorNames = [
+  'yellow',
+  'blue',
+  'green',
+  'yellowWarm',
+  'blueLight',
+  'turquoise',
```

Comment:
We have also 'pink' and 'purple' color. Is it okay to use colorNames without these colors?
Also, we have a several places in the codebase with these colors list. Probably, we can use only one place for these colors

Reply:
> Is it okay to use colorNames without these colors?

In this particular case - yes, it's ok. Because we expect this widget to have 4 items maximum.
But I can add other colors as well. Just in case.

Reply:
done